ReactWindows/ReactNative.Shared/Bridge/ReactMethodAttribute.cs

```diff
@@ -9,5 +9,8 @@ namespace ReactNative.Bridge
     [AttributeUsage(AttributeTargets.Method)]
     public sealed class ReactMethodAttribute : Attribute
     {
+        public bool IsBlockingSynchronousMethod { get; set; } = false;
+
+        public bool IsDirectCallMethod { get; set; } = false;
```

Comment:
Why do we need two properties?

Reply:
seems like direct call method is a separate feature.

---
In reply to: [173118518](https://github.com/Microsoft/react-native-windows/pull/1678#discussion_r173118518) [](ancestors = 173118518)